playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -51,6 +51,9 @@
       '{"spec":{"template":{"spec":{"containers":[{"name":"router","image":"{{ router_image }}","livenessProbe":{"tcpSocket":null,"httpGet":{"path": "/healthz", "port": 1936, "host": "localhost", "scheme": "HTTP"},"initialDelaySeconds":10,"timeoutSeconds":1}}]}}}}'
       --api-version=v1
     with_items: "{{ haproxy_routers }}"
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
Yeah, I'm suggesting that if we're going to live with the AUDIT: messages for a while we need a greppable way to denote which ones mean "we're cool, trust me" versus "fix this real soon now"